access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -163,7 +163,7 @@ public AnnotationVisitor visitAnnotation(String desc, boolean visible) {
 
                     @Override
                     public MethodVisitor visitMethod(int access, final String methodName, final String methodDesc, String signature, String[] exceptions) {
-                        return new EmptyVisitor() {
+                        return new MethodVisitor(Opcodes.ASM5) {
```

Comment:
I'm not sure to understand, in ASM 3.2 `EmptyVisitor` has [an empty implementation](http://websvn.ow2.org/blame.php?repname=asm&path=%2Ftags%2FASM_3_2%2Fsrc%2Forg%2Fobjectweb%2Fasm%2Fcommons%2FEmptyVisitor.java&rev=1389#l156) for `visitMethodInsn`. If the method was called it did nothing. In ASM 5.x `MethodVisitor.visitMethodInsn()` has a [delegation mechanism](http://websvn.ow2.org/blame.php?repname=asm&path=%2Ftags%2FASM_5_2%2Fsrc%2Forg%2Fobjectweb%2Fasm%2FMethodVisitor.java&rev=1841#l456) but it isn't used in this context, so the method also does nothing. What did I miss?